inst/@sym/horzcat.m

```diff
@@ -62,7 +62,7 @@
           'return sp.Matrix.hstack(*_proc),'
           };
 
-  varargin = sym(varargin);
+  varargin = cell_array_to_sym(varargin);
```

Comment:
The problem with `classdef` and `private` functions is a known bug that has not been fixed yet, see https://savannah.gnu.org/bugs/?45444